sormas-backend/src/main/java/de/symeda/sormas/backend/sormastosormas/rest/SormasToSormasRestClient.java

```diff
@@ -77,17 +86,33 @@ public SormasToSormasRestClient(ServerAccessDataService serverAccessDataService,
 		return sendRequest(receiverId, endpoint, body, responseType, HttpMethod.PUT);
 	}
 
-	private String buildAuthToken(OrganizationServerAccessData targetServerAccessData) {
-		String userCredentials = StartupShutdownService.SORMAS_TO_SORMAS_USER_NAME + ":" + targetServerAccessData.getRestUserPassword();
-		return "Basic " + new String(Base64.getEncoder().encode(userCredentials.getBytes(StandardCharsets.UTF_8)));
+	public <T> T get(String receiverId, String endpoint, Class<T> responseType) throws SormasToSormasException {
+		return sendRequest(receiverId, endpoint, null, responseType, HttpMethod.GET);
+	}
+
+	private String buildAuthToken(String targetId) throws SormasToSormasException {
+		String scope = String.format("s2s-%s", targetId);
+		String authToken;
+		try {
+			authToken = Oidc.requestAccessToken(
+				sormasToSormasConfig.getOidcRealmTokenEndoint(),
+				sormasToSormasConfig.getOidcClientId(),
+				sormasToSormasConfig.getOidcClientSecret(),
+				Collections.singletonList(scope));
+		} catch (Exception e) {
+			LOGGER.info(String.format("Could not requested access token for %s: %s", targetId, e));
```

Comment:
Just a suggestion, the Logger.info method supports String formatting
```java
LOGGER.info("Could not requested access token for {}: {}", targetId, e);
```

Reply:
Cool thanks!